src/test/java/com/avaje/tests/batchinsert/TestBatchInsertFlush.java

```diff
@@ -7,18 +7,22 @@
 import com.avaje.ebean.annotation.Transactional;
 import com.avaje.ebean.config.PersistBatch;
 import com.avaje.tests.model.basic.EBasicVer;
+
+import org.junit.Assume;
 import org.junit.Test;
 
 import java.sql.Timestamp;
 
 import static org.junit.Assert.assertNotNull;
 
 public class TestBatchInsertFlush extends BaseTestCase {
-
+  
   @Test
   @Transactional(batch = PersistBatch.ALL)
   public void transactional_flushOnGetId() {
 
+    Assume.assumeFalse("Skipping test because batching not yet supported for MS SQL Server.", isMsSqlServer());
```

Comment:
better use `import static org.junit.Assume.assume` - but tests should be fixed anyway.